packages/console/src/pages/ApiResourceDetails/components/DeleteForm/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { toast } from 'react-hot-toast';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import Button from '@/components/Button';
+import Card from '@/components/Card';
+import CardTitle from '@/components/CardTitle';
+import TextInput from '@/components/TextInput';
+import Close from '@/icons/Close';
+import api from '@/utilities/api';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  id: string;
+  name: string;
+  onClose: () => void;
+};
+
+const DeleteForm = ({ id, name, onClose }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  const navigate = useNavigate();
+
+  const [inputName, setInputName] = useState<string>('');
+  const [isDeleting, setIsDeleting] = useState(false);
+
+  const inputMismatched = inputName !== name;
+
+  const handleDelete = async () => {
+    try {
+      setIsDeleting(true);
+      await api.delete(`/api/resources/${id}`);
+      onClose();
+      navigate(`/api-resources`);
+      toast.success(t('api_resource_details.api_resource_deleted', { name }));
+    } catch (error: unknown) {
+      console.error(error);
+    } finally {
+      setIsDeleting(false);
+    }
+  };
+
+  return (
+    <Card className={styles.card}>
+      <div className={styles.headline}>
+        <CardTitle title="api_resource_details.reminder" />
+        <Close className={styles.close} onClick={onClose} />
+      </div>
+      <div className={styles.description}>
+        {t('api_resource_details.delete_description', { name })}
+      </div>
+      <TextInput
+        value={inputName}
+        placeholder={t('api_resource_details.enter_your_api_resource_name')}
+        hasError={inputMismatched}
+        onChange={(event) => {
+          setInputName(event.currentTarget.value);
+        }}
+      />
+      <div className={styles.line} />
```

Comment:
Could make it as a separate Component i.e. Divider

Reply:
`Divider` is not commonly used for the `line` may be implemented as a border of div with content. 
Maybe we can add the `Divider` when we use it many times.

Reply:
sorry, not following this. Barely see an individual empty div used for this case, either extract as a meaningful component or make it a border-top or border-bottom of an actual content element works.

Reply:
Add a `border` doesn't work in this case. for we use CSS to control the element's margin in this container

Reply:
Add an issue to track this. LOG-1907